### ISSUE TITLE
Not working when img src is a png extracted from blob

### ISSUE THREAD

**Issue by JL1946** [waynegm/imgViewer#21]
Hello:
I am testing your plugin for my application. It works nicely when I assign a png image as the src. 
<div class="cl_plot_view" id="id_plot_view">
        <img id="id_plot_img" src="../plots/Day_3_temp_plot.png" alt="" />
</div>
The image is a matplotlib png file.

However, when the image is sent by a websocket connection as a blob and assigned to src of the img tag, the plugin does not function. The following code is used to assign the transmitted png file:

var arrayBuffer = messageReceived.data;
                    var image = document.getElementById('id_plot_img');

                    var reader = new FileReader();
                    reader.onload = function(e) {
                        var thePlot = e.target.result;
                        image.src = thePlot;
                    };
                    reader.readAsDataURL(arrayBuffer);
                    $("#id_plot_img").imgViewer();

Google displays the following information about the element after the transfer is complete:      
![image](https://user-images.githubusercontent.com/25297557/38144532-16262784-3413-11e8-8ff7-b2abfc925732.png)

This is a picture of the webpage in Chrome (latest Version as of today):
![image](https://user-images.githubusercontent.com/25297557/38144630-93463aec-3413-11e8-9888-292e763ed8a2.png)

The Chrome console also adds:
[Violation] Added non-passive event listener to a scroll-blocking 'wheel' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952

This is a picture of the display in Firefox (latest version) when the mouse wheel is used to zoom:
![image](https://user-images.githubusercontent.com/25297557/38144692-fb489d4c-3413-11e8-8c19-8f388d70e3a2.png)
You can see that the overlay has shifted and a small icon can be seen in the upper left coner of the plot viewport.
I don't believe that the placement of the snippet ($("#id_plot_img").imgViewer();) has any effect on the result. I moved the statement around, changed it as per your demo, but the net result is the same.
Can you help with this issue?

**Comment by waynegm** [comment]
The problem is not obvious from what you have provided. Can you send a self contained example that demonstrates the problem.

**Comment by JL1946** [comment]
Thanks for getting back to me.

I tried your ImgViewer2 with the same results. Neither one produces a 
zoom/pan effect if the img src is dynamically added from a blob sent by 
the server. Both function properly if the img src is loaded from a local 
folder.

Anyway, I went in a different direction and implemented a static 2X zoom 
and pan with CSS3 and jQuery "draggable" function.which satisfies my 
requirements. Thanks for your time.

Regards,

Joe

Wayne Mogg wrote:
